tests/smoke/js/raw/inject.js

```diff
@@ -0,0 +1,21 @@
+var WebpackAssets = [assets],
```

Comment:
Why are you string replacing this assets object with a string versus for inject.js above? What is the difference between the inject.js in raw and injector.js above?